src/main/java/seedu/intrack/logic/commands/SortCommand.java

```diff
@@ -31,7 +36,8 @@ public SortCommand(String orderType) {
 
     @Override
     public CommandResult execute(Model model) {
-        if (orderType == "a") {
+        List<Internship> lastShownList = model.getSelectedInternship();
+        if (orderType.equals("a")) {
             model.ascendSort();
             model.updateFilteredInternshipList(PREDICATE_SHOW_ALL_INTERNSHIPS);
             return new CommandResult(String.format(MESSAGE_SUCCESS_A, model.getFilteredInternshipList().size()));
```

Comment:
> Use the Default Branch 
Always include a default branch in case statements.

> Furthermore, use it for the intended default action and not just to execute the last option. If there is no default action, you can use the default branch to detect errors (i.e. if execution reached the default branch, raise a suitable error). This also applies to the final else of an if-else construct. That is, the final else should mean 'everything else', and not the final option. Do not use else when an if condition can be explicitly specified, unless there is absolutely no other possibility.

The above excerpt was taken from the textbook. I have to comment it here since I am not able to comment in the line below, but in line 38, perhaps you could consider using an ```else if``` statement rather than an ```else``` statement?